docs/docs/cmd/entra/group/group-set.mdx

```diff
@@ -0,0 +1,99 @@
+import Global from '/docs/cmd/_global.mdx';
+
+# entra group set
+
+Updates a Microsoft Entra group
+
+## Usage
+
+```sh
+m365 entra group set [options]
+```
+
+## Options
+
+```md definition-list
+`-i, --id [id]`	
+: The ID of the Microsoft Entra group to update. Specify either `id` or `displayName` but not both.
+
+`-n, --displayName [displayName]`	
+: The display name of the Microsoft Entra group to update. Specify either `id` or `displayName` but not both.
+
+`--newDisplayName [newDisplayName]`	
+: The new display name of the Microsoft Entra group. The maximum length is 256 characters.
+
+`--description [description]`
+: The new description for the group.
+
+`--mailNickname [mailNickname]`
+: The new mail alias for the group (part before the @). Use only for mail-enabled groups. Maximum length is 64 characters.
+
+`--ownerIds [ownerIds]`
+: Comma-separated list of IDs of Microsoft Entra users that will be the group owners. Specify either `ownerIds` or `ownerUserNames`, but not both.
+
+`--ownerUserNames [ownerUserNames]`
+: Comma-separated list of UPNs of Microsoft Entra users that will be the group owners. Specify either `ownerIds` or `ownerUserNames`, but not both.
+
+`--memberIds [memberIds]`
+: Comma-separated list of IDs of Microsoft Entra users that will be the group members. Specify either `memberIds` or `memberUserNames`, but not both.
+
+`--memberUserNames [memberUserNames]`
+: Comma-separated list of UPNs of Microsoft Entra users that will be the group members. Specify either `memberIds` or `memberUserNames`, but not both.
+
+`--visibility [visibility]`
+: Specifies the group join policy and group content visibility for Microsoft 365 groups. Possible values are: `Private` or `Public`. Specify only when targeting a Microsoft 365 group.
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+The `visibility` option affects the behavior of the group.
+
+With the `Public` visibility:
+- Anyone can join the group without needing owner approval. 
+- Anyone can view the attributes of the group. 
+- Anyone can see the members of the group.
+
+With the `Private` visibilty:
+- Owner approval is needed to join the group.
+- Anyone can view the attributes of the group.
+- Anyone can see the members of the group.
+
+:::
```

Comment:
It's good that you strive for consistency, but in this case, I don't think this info has to be highlighted in an admonition. To me, it's not that important, and for most people quite obvious. So in my opinion, just a regular remark wil suffice. You can update this as well in the `add` command.